region/maridia/inner-yellow/Bug Sand Hole.json

```diff
@@ -492,6 +492,10 @@
           ]}
         ]},
         {"doorUnlockedAtNode": 1}
+      ],
+      "unlocksDoors": [
+        {"nodeId": 1, "types": ["super"], "requires": []},
+        {"nodeId": 1, "types": ["missiles", "powerbomb"], "requires": ["never"]}
```

Comment:
these seem fine to me with gravity or hijump. I could see it being added with a careful jump and/or 1-2 Zoa hits